docs/DeveloperGuide.md

```diff
@@ -1,29 +1,215 @@
 # Developer Guide
+It is recommended that you read through the user guide to familiarize yourself with the program before using the 
+developer guide.
+
+# Setting up PaperTrade
+Fork the repository and clone the fork into your computer.
+
+To set up the project:
+1. Ensure that you have JDK 11 installed.
+2. Import the project as a Gradle project.
+3. To build the project, run "./gradlew build" on a Unix machine or run "gradlew build" on Windows.
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+### Architecture
+
+![](./diagrams/Packages.png)
+
+The Package Diagram above gives a high level view of the project structure and the classes in each package. 
+
+![](./diagrams/Architecture.png)
```

Comment:
Will it be better to put the high level design of PaperTrade above the project structure?